### PR TITLE
[misc] add quantiles to scale test

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: notebook/scale-test.py [hunk 1]
```diff
@@ -130,6 +130,7 @@ def get_cookie(session, name):
 
     print(f'successes: {len(times)} / {n} = {len(times) / n}')
     print(f'mean time: {sum(times) / n}')
+    print(f'quantiles 50%/95%/99%:: {np.quantile(times, [.5, .95, .99])}')
     print(f'max time: {max(times)}')
```
Comment (cseed) [inline notebook/scale-test.py]: Want to include max with the quantiles?  50/95/99/max?

Comment (tpoterba) [reply notebook/scale-test.py]: good idea, done

File: notebook/scale-test.py [hunk 2]
```diff
@@ -130,7 +130,7 @@ def get_cookie(session, name):
 
     print(f'successes: {len(times)} / {n} = {len(times) / n}')
     print(f'mean time: {sum(times) / n}')
-    print(f'max time: {max(times)}')
+    print(f'quantiles min/50%/95%/99%/max:: {np.quantile(times, [0.0, .5, .95, .99, 1.0])}')
```
Comment (cseed) [inline notebook/scale-test.py]: Did you mean to have the double colon?

Comment (tpoterba) [reply notebook/scale-test.py]: no, my keyboard inserts between 0 and 3 characters on each key press.

**Pull request merged**